newscoop/library/Newscoop/Entity/User.php

```diff
@@ -1227,12 +1238,18 @@ public function getView()
 
     public function serialize()
     {
-        return serialize($this->id);
+        return serialize(array(
```

Comment:
not sure if it will not break some other parts where we just echo user object and get his id.